server/app/controllers/applicant/ApplicantProgramsController.java

```diff
@@ -88,7 +95,9 @@ public CompletionStage<Result> index(Request request, long applicantId) {
             ex -> {
               if (ex instanceof CompletionException) {
                 if (ex.getCause() instanceof SecurityException) {
-                  return unauthorized();
+                  // If the applicant id in the URL does not correspond to the current user, start
```

Comment:
**suggestion:** We have a comment here, but not for similar case below. Maybe make these consistent?

Reply:
Done